p2p/pkg/node/node.go

```diff
@@ -459,7 +459,7 @@ func NewNode(opts *Options) (*Node, error) {
 					Startable: depositMgr.(*depositmanager.DepositManager),
 				},
 			)
-			preconfEncryptor, err := preconfencryptor.NewEncryptor(opts.KeySigner, keysStore)
+			preconfEncryptor, err := preconfencryptor.NewEncryptor(opts.KeySigner, keysStore, chainID, opts.PreconfContract)
```

Comment:
Do we check the chainID against the deployed version. E.g should we have this as a static part of our contract as well. So we can check against it when we verify the signature in the preconfs contract

Reply:
We don’t check because, if they are different, the signature will be automatically rejected since the domain separators will not match